firebase.json

```diff
@@ -22,6 +22,11 @@
         "source": "/docs/kovan-keeper-network-beta/",
         "destination": "/docs/chainlink-keepers/introduction/",
         "type": 301
+      },
+      {
+        "source": "/docs/adapters",
```

Comment:
Can you make sure these end with `/`?

Reply:
Completed and squashed changes into one commit.